content/4_Gold/Combinatorics.mdx

```diff
@@ -868,28 +868,41 @@ non-empty. Thus, the answer is obtained by the second formula: $\binom{n-1}{k-1}
 ```cpp
 #include <iostream>
 
-/** @return n choose k, computed naively since the problem has no overflow */
+using namespace std;
+
 long long comb(int n, int k) {
 	if (k > n - k) { k = n - k; }
-	long long ret = 1;
-	for (int i = 0; i < k; i++) {
-		// this is done instead of *= for divisibility issues
-		ret = ret * (n - i) / (i + 1);
-	}
-	return ret;
+	long long ans = 1;
+	for (int i = 0; i < k; i++) { ans = ans * (n - i) / (i + 1); }
+	return ans;
 }
```

Comment:
uh why'd you undo the changes

Reply:
not sure where that came from

Reply:
kk i'll fix it myself sometime

Reply:
the bot is also doing stuff like that

Reply:
stuff like what

Reply:
like that for loop changing from multiple lines to one

maybe i should pull code every time i edit

Reply:
no yeah we just have to live with that

> maybe i should pull code every time i edit

yeah this is also good